ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/PresentmentDetails.kt

```diff
@@ -0,0 +1,14 @@
+package com.braintreepayments.api.shopperinsights
+
+/**
+ * Initializes a new PresentmentDetails instance
+ *
+ * @param type An ExperimentType that is either a control or test type
+ * @param buttonOrder optional Represents this buttons order in context of other buttons.
+ * @param pageType optional Represents the page or view the button is rendered on.
```

Comment:
```suggestion
 * @property type An ExperimentType that is either a control or test type
 * @property buttonOrder optional Represents this buttons order in context of other buttons.
 * @property pageType optional Represents the page or view the button is rendered on.
```

We should use the property annotation for class properties.